src/index.html

```diff
@@ -17,11 +17,390 @@
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="product-card__reviews-text">Reviews: 5</p>
+          </div>
+
+          <div class="product-card__price">
+            <p class="product-card__price-label">Price:</p>
+            <p class="product-card__price-value">$2,199</p>
+          </div>
+          <button
+            type="button"
+            class="product-card__buy-button"
+          >
+            BUY
+          </button>
+        </div>
+
+        <div class="product-card">
+          <div class="product-card__image-container">
+            <img
+              src="./images/imac.jpeg"
+              alt="imac"
+              class="product-card__image"
+            />
+          </div>
+          <h1 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+          <h2 class="product-card__code">Product code: 195434</h2>
+
+          <div class="product-card__reviews">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="product-card__reviews-text">Reviews: 5</p>
+          </div>
+
+          <div class="product-card__price">
+            <p class="product-card__price-label">Price:</p>
+            <p class="product-card__price-value">$2,199</p>
+          </div>
+          <button
+            type="button"
+            class="product-card__buy-button"
+          >
+            BUY
+          </button>
+        </div>
+
+        <div class="product-card">
+          <div class="product-card__image-container">
+            <img
+              src="./images/imac.jpeg"
+              alt="imac"
+              class="product-card__image"
+            />
+          </div>
+          <h1 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+          <h2 class="product-card__code">Product code: 195434</h2>
+
+          <div class="product-card__reviews">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="product-card__reviews-text">Reviews: 5</p>
+          </div>
+
+          <div class="product-card__price">
+            <p class="product-card__price-label">Price:</p>
+            <p class="product-card__price-value">$2,199</p>
+          </div>
+          <button
+            type="button"
+            class="product-card__buy-button"
+          >
+            BUY
+          </button>
+        </div>
```

Comment:
The product card details are repeated multiple times with the same content. If this repetition is intentional for demonstration purposes, it is fine. However, if the content is meant to be dynamic or different for each card, consider using a template engine or JavaScript to dynamically generate these cards with varying content.